toutoumomoma.go

```diff
@@ -221,29 +203,11 @@ func (f *File) ImportHash() (hash []byte, imports []string, err error) {
 // If the file is an executable, but not a gc-compiled Go executable, ErrNotGoExecutable
 // will be returned.
 func (f *File) GoSymbolHash(stdlib bool) (hash []byte, imports []string, err error) {
-	ok, err := f.isGoExecutable()
-	if !ok || err != nil {
-		if err != nil {
-			return nil, nil, err
-		}
-		return nil, nil, ErrNotGoExecutable
-	}
+	return f.goSymbolHash(stdlib)
+}
 
-	imports, err = f.goSymbols(stdlib)
-	if err != nil {
-		return nil, nil, err
-	}
-	h := md5.New()
-	if len(imports) == 0 {
-		return h.Sum(nil), nil, nil
-	}
-	for i, imp := range imports {
-		if i != 0 {
-			_, _ = h.Write([]byte{','})
-		}
-		fmt.Fprint(h, imp)
-	}
-	return h.Sum(nil), imports, nil
+func (f *File) GoSymbols(stdlib bool) (imports []string, err error) {
```

Comment:
Similar comment as above. The non-fips implementation of `func (f *File) goSymbolHash(stdlib bool) (hash []byte, imports []string, err error)` doesn't use it either.